### ISSUE TITLE
Rule idea: disallow nesting inside a mixin

### ISSUE THREAD

**Issue by paazmaya** [stylelint/stylelint#1831]
A new rule that would disallow using nesting inside a mixin.

Might be quite specific but as more and more preprosessors are being supported, why not.

For example this kind of CSS would trigger warning:

```css
@define-mixin gansta () {
  border: inherit;
  & > div {
    background-color: red;
  }
}
```

**Comment by evilebottnawi** [comment]
@paazmaya we create rules only for standard `css` and future standard `css`, not `less`, `scss` and etc. Use plugin for this feature or we can create `stylelint-less` as https://github.com/kristerkari/stylelint-scss

**Comment by paazmaya** [comment]
Thank you for the clarification.
